assets/dev/scss/admin/_experiments.scss

```diff
@@ -34,4 +33,8 @@
 		font-weight: bold;
 		font-style: italic;
 	}
+
+	&__button {
+		@include margin(20px, 5px, 18px, 0 !important);
+	}
```

Comment:
Please change the selector to be stronger than the WordPress selectors and remove the `!important`.